app/hlds/definitions.py

```diff
@@ -15,7 +17,11 @@
 DATA_DIR = ROOT_DIR / "menus"
 
 location_definitions: List[Location] = parse_all_directory(str(DATA_DIR))
-location_definitions.sort(key=lambda l: l.name)
+
+def extend_locations_with_osm(app) -> None:
+    global location_definitions
```

Comment:
:thinking:

Reply:
die global moet daar staan of dat breekt mijn code